kernel/setup/steps/ezstep_installer.php

```diff
@@ -489,12 +489,11 @@ function checkDatabaseRequirements( $dbCharset = false, $overrideDBParameters =
             $checkedCharset = $db->checkCharset( $charsetsList, $currentCharset );
             if ( $checkedCharset === false )
             {
-                // If the current charset is utf-8 we use that instead
-                // since it can represent any character possible in the chosen languages
-                if ( $currentCharset == 'utf-8' )
+                // If the current charset is utf-8 or utf8mb4 we use that instead
+                // since they can represent any character possible in the chosen languages
```

Comment:
Disclaimer: Not 100% true, certain exotic natural language characters are not possible in utf-8, but in most cases the reason utf8mb4 is needed is security and emoji.